schema/deploy/warehouse/identifier/triggers/barcode-distance-check.sql

```diff
@@ -31,8 +37,9 @@ create or replace function warehouse.identifier_barcode_distance_check() returns
 
         select barcode into conflicting_barcode from (
             select barcode
-              from warehouse.identifier
-             where hamming_distance_lte(lower(barcode), new_barcode, minimum_distance-1) < minimum_distance
+            from warehouse.identifier
+            where slices && NEW.slices
+            and hamming_distance_lte(lower(barcode), new_barcode, minimum_distance-1) < minimum_distance
```

Comment:
Minor nit (which is not merge-blocking): this diff changes the formatting unnecessarily. As a general rule, it's nice to "when in Rome…" by staying consistent with existing formatting in a file, since it minimizes diffs and avoids mixed styles within a single file.

Reply:
I realize that but could not in clear conscious leave the lines unaligned since I was touching one of them.

Reply:
I've never seen a FROM indented more than a SELECT in SQL anywhere, but I've restored this back to how it was formatted in the fixup commit.

Reply:
Ah, I swear it's a style used by people other than me. :-) In any case, it's one [prevailing style within ID3C](https://github.com/search?q=select+OR+where++repo%3Aseattleflu%2Fid3c+language%3APython+language%3APython&type=Code&ref=advsearch&l=Python&l=Python).